articles/quickstart/backend/aspnet-core-webapi/files/configure-middleware.md

```diff
@@ -33,10 +29,18 @@ public void ConfigureServices(IServiceCollection services)
     services.AddSingleton<IAuthorizationHandler, HasScopeHandler>();
 }
 
-public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+builder.Services.AddAuthorization(options =>
```

Comment:
We should remove the code above this one.

Reply:
Done. Had a duplicate code block. Removed that.